spark/src/main/java/org/apache/comet/CometBatchIterator.java

```diff
@@ -33,12 +33,27 @@
 public class CometBatchIterator {
   final Iterator<ColumnarBatch> input;
   final NativeUtil nativeUtil;
+  private ColumnarBatch currentBatch = null;
 
   CometBatchIterator(Iterator<ColumnarBatch> input, NativeUtil nativeUtil) {
     this.input = input;
     this.nativeUtil = nativeUtil;
   }
 
+  /**
+   * Fetch the next input batch.
+   *
+   * @return Number of rows in next batch or -1 if no batches left.
+   */
+  public int hasNext() {
```

Comment:
See the earlier comment about returning a bool from this method.

Reply:
Oh. Ignore my previous comment. Why would changing the return type of `hasNext` not work?

Reply:
I run into this error when trying to return bool:

```
224 |         comet_batch_iterator(iter).has_next() -> bool)?
    |                                                  ^^^^ the trait `std::convert::From<JValueGen<JObject<'_>>>` is not implemented for `bool`, which is required by `bool: TryFrom<JValueGen<JObject<'_>>>`
```